test/CMakeLists.txt

```diff
@@ -20,9 +20,9 @@ enable_testing()
 
 if(MSVC)
   # VS needs -C config to run correct
-  add_custom_target(tests "${CMAKE_CTEST_COMMAND}" -C $<$<CONFIG:Debug>:Debug>$<$<CONFIG:Release>:Release> -V -T Test VERBATIM)
+  add_custom_target(tests "${CMAKE_CTEST_COMMAND}" -C $<$<CONFIG:Debug>:Debug>$<$<CONFIG:Release>:Release> -E gpu_octree -V -T Test VERBATIM)
```

Comment:
Add a comment for both lines indicating that you're excluding gpu_octree because CI does not have access to gpu hardware to run the tests.